packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

```diff
@@ -0,0 +1,262 @@
+* `${clientId}`: Your customer's integration client ID
+* `${clientSecret}`: Your customer's integration client secret
+* `${scopes}`: The resource scopes required for the access token
+
+> **Note:** If you're using an OAuth 2.0 library, you typically need to you configure an OAuth client class with a `tokenUri` parameter, as well as the `clientId` and `clientSecret` parameters. Specify the `tokenUri` string as `https://${customerOktaDomain}/oauth2/v1/token`.
+
+The following is the process for using a Basic authorization header for the `/token` request:
+
+1. Base64 encode the string and set it in the authorization header:
+
+   ```sh
+   Authorization: Basic ${Base64(${clientId}:${clientSecret})}
+   ```
+
+1. Make a request to the [/token](/docs/reference/api/oidc/#token) endpoint with these query parameters:
+   * `grant_type` set to `client_credentials`
+   * `scope` must be at least one of the [Okta API scopes](/docs/guides/implement-oauth-for-okta/main/#scopes-and-supported-endpoints)
+
+   Request example:
+
+   ```bash
+   curl --request POST \
+     --url https://${customerOktaDomain}/oauth2/v1/token \
+     --header 'Accept: application/json' \
+     --header 'Authorization: Basic MG9hY...' \
+     --header 'Cache-control: no-cache' \
+     --header 'Content-type: application/x-www-form-urlencoded' \
+     --data 'grant_type=client_credentials&scope=okta.users.read okta.groups.read'
+   ```
+
+1. If the request is successful, the token is returned in the body of the response.
+
+   Successful response example:
+
+   ```json
+   {
+      "token_type": "Bearer",
+      "expires_in": 3600,
+      "access_token": "eyJraWQiOiJ.....UfThlJ7w",
+      "scope": "okta.users.read okta.groups.read"
+   }
+   ```
+
+   Error response example:
+
+   ```json
+   {
+      "error": "invalid_client",
+      "error_description": "The client secret supplied for a confidential client is invalid."
+   }
+   ```
+
+   See the [list of token endpoint errors](/docs/reference/api/oidc/#possible-errors-4).
+
+> **Note:** By default, the access token expires in an hour. You need to request for a new access token after your token expires.
+
+### Make Okta API requests
+
+When you have an access token, you can use it to make requests to the [Okta management APIs](/docs/reference/core-okta-api/). Set the access token as a bearer token in an authorization header.
+
+Example request:
+
+```bash
+curl -X GET "https://${customerOktaDomain}/api/v1/users"
+    -H "Accept: application/json"
+    -H "Authorization: Bearer ${accessToken}"
+```
+
+### Test your API service flow
+
+Use the credentials from your test integration (Okta domain, client ID and secret) to test your service app. You can use the [interface that you build](#collect-and-save-customer-credentials) or enter the credentials manually to configure your service app. Trigger a process from your service app to access Okta endpoints and verify that the process works as intended.
```

Comment:
```suggestion
Use the credentials from your test integration (Okta domain, client ID, and secret) to configure and test your service app. Trigger a process from your service app to access Okta endpoints and verify that the process works as intended.
```

Reply:
keeping original, since I think this extra info is important.